RELEASE.md

```diff
@@ -15,7 +15,7 @@ Please follow the established format:
 
 ## Bug fixes and other changes
 
-<!-- Add release notes for the upcoming release here -->
+* Fix `%run_viz` line magic to display kedro viz inside Jupyter notebook. Note that the fix needs `kedro>=0.16.2` to take effect.
```

Comment:
This is super minor, just to clarify that if they're not on 0.16.* already they're alright. But I don't have a strong stance on it.
```suggestion
* Fix `%run_viz` line magic to display kedro viz inside Jupyter notebook, previously broken for kedro 0.16.0/0.16.1. Note that the fix needs `kedro>=0.16.2` to take effect.
```